include/z64item.h

```diff
@@ -145,46 +145,46 @@ typedef enum {
 } InventorySlot;
 
 typedef enum {
-    /* 0x00 */ ITEM_STICK,
-    /* 0x01 */ ITEM_NUT,
+    /* 0x00 */ ITEM_DEKU_STICK,
+    /* 0x01 */ ITEM_DEKU_NUT,
     /* 0x02 */ ITEM_BOMB,
     /* 0x03 */ ITEM_BOW,
     /* 0x04 */ ITEM_ARROW_FIRE,
     /* 0x05 */ ITEM_DINS_FIRE,
     /* 0x06 */ ITEM_SLINGSHOT,
     /* 0x07 */ ITEM_OCARINA_FAIRY,
-    /* 0x08 */ ITEM_OCARINA_TIME,
+    /* 0x08 */ ITEM_OCARINA_OF_TIME,
     /* 0x09 */ ITEM_BOMBCHU,
     /* 0x0A */ ITEM_HOOKSHOT,
     /* 0x0B */ ITEM_LONGSHOT,
     /* 0x0C */ ITEM_ARROW_ICE,
     /* 0x0D */ ITEM_FARORES_WIND,
     /* 0x0E */ ITEM_BOOMERANG,
-    /* 0x0F */ ITEM_LENS,
-    /* 0x10 */ ITEM_BEAN,
+    /* 0x0F */ ITEM_LENS_OF_TRUTH,
+    /* 0x10 */ ITEM_MAGIC_BEANS,
```

Comment:
Could you explain the reasoning behind making this one plural when all the other "ammo"-type items are singular? (eg. stick or bomb)

Reply:
It wasn't a conscious decision, I guess just like you say "use deku stick here" you naturally say "use magic beans here" and not "use magic bean here"

Note I must not be alone since `icon_item_static` also has it as plural `gMagicBeansIconTex`

But I'm fine changing all those to singular "magic bean", it would indeed make more sense

Reply:
Well that's just what the items are called, see the textures in `item_name_static`.

Reply:
singular does seem more consistent with the other ammos.

Reply:
I'll change to singular